_playwright-tests/test_system.test.ts

```diff
@@ -3,7 +3,7 @@ import { test, navigateToInventorySystemsFunc } from './helpers/navHelpers';
 import { uploadArchive } from './helpers/uploadArchive';
 import { searchByName } from './helpers/filterHelpers';
 
-test.skip('User should be able to edit and delete a system', async ({ page }) => {
+test('User should be able to edit and delete a system', async ({ page }) => {
```

Comment:
Please let this test case skip for now. We need to modify the upload archive function